taskManager/app/controllers/tasks_controller.rb

```diff
@@ -2,10 +2,14 @@ class TasksController < ApplicationController
   before_action :task, only: %i[destroy show edit update]
 
   ORDER = %w[asc desc].freeze
+  PER = 5,freeze
 
   def index
     @search_params = task_search_params
-    @tasks = Task.search(@search_params).order format('%s %s', sort_position, sort_order)
+    @tasks = Task.page(params[:page])
+      .per(PER)
+      .search(@search_params)
+      .order format('%s %s', sort_position, sort_order)
```

Comment:
```suggestion
      .order(format('%s %s', sort_position, sort_order))
```
前回のレビューでは見逃してたのですが、orderの引数は`()`でくくってあげた方が分かりやすいです